platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ImportMigrationManagerImpl.java

```diff
@@ -123,6 +120,8 @@ public ImportMigrationManagerImpl(
       // process migratables' metadata
       JsonUtils.getMapFrom(metadata, MigrationContextImpl.METADATA_MIGRATABLES)
           .forEach((id, o) -> getContextFor(id).processMetadata(JsonUtils.convertToMap(o)));
+    } catch (FileNotFoundException e) {
```

Comment:
❓ Which part of the code now throws this FileNotFoundException? It used to be thrown below when the zip file was created which you are now doing when you are creating the MigrationZipFile which is done before we get here.